test/backgroundprocessor/BackgroundBlurProcessor.test.ts

```diff
@@ -690,6 +690,12 @@ describe('BackgroundBlurProcessor', () => {
         const supportedUserAgents = [
           'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3865.75 Safari/537.36',
           'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Safari/605.1.15',
+          'Mozilla/5.0 (iPhone; CPU iPhone OS 16_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko)' +
```

Comment:
This line wrap is weird, I don't think we need to use `+` to concatenate string here.